### PR TITLE
introduce conference= query param

### PR DESCRIPTION
**Pull request by martinbonnin**
The GraphQL API can now query data from different conferences using the `conference=` query param. Right now it supports `droidconsf|devfestnantes`

This PR has a couple of breaking change in the GraphQL API:

* `speaker.bio` is now nullable
* session can now have several rooms (or no room)
* `session.description` is now nullable
* `SessionEdge` doesn't exist anymore because [DataStore](https://cloud.google.com/datastore) doesn't have a concept of per-entity cursor. Only per-query. This probably broke the existing pagination PR. Sorry @BoD 😬 

It's non-breaking for apps in the wild because the url changed from 

```
https://confetti-349319.uw.r.appspot.com/graphql
```
to 

```
https://graphql-dot-confetti-349319.uw.r.appspot.com/graphql[?conference=droidconsf]
```

I'll decommission the old url in a few (that'd be a nice use case for studio actually :) )

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </JetCodeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+      <indentOptions>
+        <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      </indentOptions>
+    </codeStyleSettings>
```
Comment (martinbonnin) [inline .idea/codeStyles/Project.xml:11]: I "guessed" this from the other files. This puts indent as 4 spaces. I usually tend to like smaller indents better but 4 spaces work too :) 

0_rWvhTR1T46HOPSzO

File: androidApp/src/main/java/dev/johnoreilly/confetti/ConfettiViewModel.kt [hunk 2]
```diff
@@ -23,7 +23,7 @@ class ConfettiViewModel(private val repository: ConfettiRepository): ViewModel()
 
     fun getSessionSpeakerLocation(session: SessionDetails): String {
         var text = session.speakers.joinToString(", ") { it.name }
-        text += " / ${session.room.name} / ${getLanguageInEmoji(session.language)}"
+        text += " / ${session.room?.name} / ${getLanguageInEmoji(session.language)}"
```
Comment (martinbonnin) [inline androidApp/src/main/java/dev/johnoreilly/confetti/ConfettiViewModel.kt:26]: Unfortunate consequence of some sessions not having a room (like lunch). Ideally, there is some UI for that

**Comment by joreilly** [comment]
Nice!

File: backend/service-graphql/src/main/appengine/app.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+service: graphql
+runtime: java11
+instance_class: F2
```
Comment (martinbonnin) [inline backend/service-graphql/src/main/appengine/app.yaml:3]: Instances need 512MB of RAM now... Not sure why. Maybe the DataStore client libraries...

**Comment by joreilly** [comment]
@martinbonnin feel free of course to merge this whenever you're ready to.

**Pull request merged**